curriculum/challenges/english/07-scientific-computing-with-python/learn-special-methods-by-building-a-vector-space/65f0694fb2296f3caadf8347.md

```diff
@@ -0,0 +1,45 @@
+---
+id: 65f0694fb2296f3caadf8347
+title: Step 6
+challengeType: 20
+dashedName: step-6
+---
+
+# --description--
+
+The length of a vector `a`, or norm, is tipically indicated as \\( \| \mathbf{a} \| \\). Applying the Pythagorean theorem, it can be calculated as the square root of the sum of its squared components:
```

Comment:
```suggestion
The length of a vector `a`, or norm, is typically indicated as \\( \| \mathbf{a} \| \\). Applying the Pythagorean theorem, it can be calculated as the square root of the sum of its squared components:
```